.github/workflows/staticcheck.yaml

```diff
@@ -103,14 +197,3 @@ jobs:
         with:
           sarif_file: 'trivy-health-probe-proxy-linux-results.sarif'
           category: health-probe-proxy-linux-image
-
-      - name: Run Trivy vulnerability scanner in repo mode
```

Comment:
And why making such huge change to github pipelines? I think we should scan the whole repo

Reply:
please see line 65

All these actions share two steps: checkout and setup go.
It will save time.